build/scripts/darwin_amd64.sh

```diff
@@ -13,7 +13,9 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-set -eu
+set -eux
+
+CURRENT_TAG=$(git tag --sort v:refname | tail -n 1)
```

Comment:
Can you add a version.txt in this build/scripts directory, set the content to 0.3.2, and "cat" it in the build scripts? (Originally I thought about plumbing through a versions number from the golang client API, but decided that the version thing is only meaningful for a built binary.) Thanks!

Reply:
Done!